.github/workflows/ci.yml

```diff
@@ -97,16 +98,28 @@ jobs:
             test: test_multi_sessionjob.sh
           - namespace: default
             test: test_autoscaler.sh
+          - namespace: default
+            test: test_dynamic_config.sh
           - mode: standalone
             test: test_autoscaler.sh
+          - mode: standalone
+            test: test_dynamic_config.sh
           - version: v1_13
             test: test_autoscaler.sh
+          - version: v1_13
+            test: test_dynamic_config.sh
           - version: v1_14
             test: test_autoscaler.sh
+          - version: v1_14
+            test: test_dynamic_config.sh
           - version: v1_15
             test: test_autoscaler.sh
+          - version: v1_15
+            test: test_dynamic_config.sh
           - version: v1_16
             test: test_autoscaler.sh
+          - version: v1_16
+            test: test_dynamic_config.sh
```

Comment:
We should also exclude dynamic for 1.17 (only keep for 1.18)

Reply:
Make sense.